web/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
   buildNumber: 0,
   appVersion: 0,
   deviceName: '',
-  userAgent: window.navigator.userAgent,
+  userAgent: () => new Promise((resolve, reject) => resolve(window.navigator.userAgent)),
```

Comment:
This could be just `() => Promise.resolve(window.navigator.userAgent)` which avoids creating an extra closure :)